src/components/form/form-summary/form-summary.style.js

```diff
@@ -0,0 +1,49 @@
+import styled, { css } from 'styled-components';
+import baseTheme from '../../../style/themes/base';
+
+const StyledFormSummary = styled.div`
+  display: inline-flex;
+  align-items: center;
+  font-size: 13px;
+  font-weight: 700;
+  margin: -8px;
+  white-space: nowrap;
+  padding: 8px;
+  border-radius: 4px;
+ 
+  ${({ isInvalid, theme }) => isInvalid && css`
+    background-color: ${theme.form.invalid};
+    margin-left: 15px;
+  `}
+`;
+
+export const StyledInternalSummary = styled.span`
+  padding: 0 3px;
+
+  carbon-form-summary__icon {
+    padding: 0 3px;
+    position: relative;
+    top: -2px;
+  }
+  ${({ type }) => type === 'warning' && css`color: #FF7D00;`}
+  ${({ type }) => type === 'error' && css`color: #D63F40;`}
```

Comment:
why are these colors hardcoded?

Reply:
I'm waiting for some direction from UX, but if I'll try playing with the palette object and get as close as possible

Reply:
After discussing with UX I have implemented the colors as base error and warning already defined in the theme object